ios/RNTPTDocumentView.m

```diff
@@ -2336,8 +2336,7 @@ - (void)applyViewerSettings:(PTDocumentBaseViewController *)documentViewControll
     }
 
     // Enable/disable restoring state (last read page).
-    [NSUserDefaults.standardUserDefaults setBool:self.saveStateEnabled
-                                          forKey:@"gotoLastPage"];
+    documentViewController.viewStatePersistenceEnabled = self.saveStateEnabled;
```

Comment:
These do different things, both of which are needed for different flows:
```suggestion
    [NSUserDefaults.standardUserDefaults setBool:self.saveStateEnabled
                                           forKey:@"gotoLastPage"];
    documentViewController.viewStatePersistenceEnabled = self.saveStateEnabled;
```

Reply:
Fixed @dluco